pkg/jira/jira.go

```diff
@@ -101,6 +123,8 @@ func (c *Verifier) verifyExtPRs(issue *jiraBaseClient.Issue, extPRs []pr, errs [
 				errs = append(errs, newErr)
 				return false, "", errs, false
 			}
+			// Comment on the PR saying that this PR is included in the release
+			_, success = c.commentOnPR(extPR, message)
```

Comment:
We should not override the `success` bool here, since it is used in a condition check later on.